### ISSUE TITLE
JSON Schema: Property $schema is not allowed

### ISSUE THREAD

**Issue by KubaJastrz** [JamieMason/syncpack#236]
## Description

When I try to use `$schema` in `.syncpackrc.json` file, I get this error:

![ishare-1721746708](https://github.com/user-attachments/assets/b354520d-adae-418c-adf7-551f9be04919)

```
Property $schema is not allowed.
```

It happens for relative paths (to node_modules), as well as http links (to unpkg, as described in [the documentation](https://jamiemason.github.io/syncpack/integrations/json-schema/)).

## Suggested Solution

Add this to schema (copied from [json-schema](https://json-schema.org/draft-07/schema#)):

```json
        "$schema": {
            "type": "string",
            "format": "uri"
        },
```

## Help Needed

I can probably create a PR if you want.

**Comment by KubaJastrz** [comment]
Tested version: 12.3.2

**Comment by KubaJastrz** [comment]
duplicate of #207

**Comment by JamieMason** [comment]
Released in [12.4.0](https://github.com/JamieMason/syncpack/releases/tag/12.4.0)
